mlir/lib/Dialect/Linalg/IR/LinalgOps.cpp

```diff
@@ -5243,6 +5243,26 @@ LogicalResult UnPackOp::canonicalize(UnPackOp unPackOp,
                              [&]() { unPackOp.setDpsInitOperand(0, newDest); });
     return success();
   }
+  /// extract_slice(unpack(x)) -> unpack(x)
```

Comment:
The comment is not clear enough to me because there is a `destination` in unpack. How about forming it this way?

```
/// extract_slice(unpack(x into y)) -> unpack(x into extract_slice(y))
```